client/lib/ScriptInstance.ts

```diff
@@ -1,15 +1,16 @@
-import { v1 as uuidv1 } from 'uuid';
+import { nanoid } from 'nanoid';
 import IScriptInstanceMeta from '@ulixee/hero-interfaces/IScriptInstanceMeta';
 
 export default class ScriptInstance {
-  public readonly id: string = uuidv1();
+  public readonly id: string = nanoid();
```

Comment:
Why switch from uuid to nanoid? I'm not against it; just curious.

Reply:
ids are shorter and seem to sort better in the file system